src/AppInstallerCLICore/Command.h

```diff
@@ -56,16 +56,18 @@ namespace AppInstaller::CLI
         };
 
         Command(std::string_view name, std::string_view parent) :
-            Command(name, parent, Settings::ExperimentalFeature::Feature::None) {}
-        Command(std::string_view name, std::string_view parent, Command::Visibility visibility) :
-            Command(name, parent, visibility, Settings::ExperimentalFeature::Feature::None) {}
-        Command(std::string_view name, std::string_view parent, Settings::ExperimentalFeature::Feature feature) :
-            Command(name, parent, Command::Visibility::Show, feature) {}
-        Command(std::string_view name, std::string_view parent, Settings::TogglePolicy::Policy groupPolicy) :
-            Command(name, parent, Command::Visibility::Show, Settings::ExperimentalFeature::Feature::None, groupPolicy) {}
-        Command(std::string_view name, std::string_view parent, Command::Visibility visibility, Settings::ExperimentalFeature::Feature feature) :
-            Command(name, parent, visibility, feature, Settings::TogglePolicy::Policy::None) {}
-        Command(std::string_view name, std::string_view parent, Command::Visibility visibility, Settings::ExperimentalFeature::Feature feature, Settings::TogglePolicy::Policy groupPolicy);
+            Command(name, {}, parent) {}
+        Command(std::string_view name,std::vector<std::string_view> aliases, std::string_view parent) :
+            Command(name, aliases, parent, Settings::ExperimentalFeature::Feature::None) {}
+        Command(std::string_view name,std::vector<std::string_view> aliases, std::string_view parent, Command::Visibility visibility) :
+            Command(name, aliases, parent, visibility, Settings::ExperimentalFeature::Feature::None) {}
+        Command(std::string_view name,std::vector<std::string_view> aliases, std::string_view parent, Settings::ExperimentalFeature::Feature feature) :
+            Command(name, aliases, parent, Command::Visibility::Show, feature) {}
+        Command(std::string_view name,std::vector<std::string_view> aliases, std::string_view parent, Settings::TogglePolicy::Policy groupPolicy) :
+            Command(name, aliases, parent, Command::Visibility::Show, Settings::ExperimentalFeature::Feature::None, groupPolicy) {}
+        Command(std::string_view name,std::vector<std::string_view> aliases, std::string_view parent, Command::Visibility visibility, Settings::ExperimentalFeature::Feature feature) :
+            Command(name, aliases, parent, visibility, feature, Settings::TogglePolicy::Policy::None) {}
+        Command(std::string_view name,std::vector<std::string_view> aliases, std::string_view parent, Command::Visibility visibility, Settings::ExperimentalFeature::Feature feature, Settings::TogglePolicy::Policy groupPolicy);

```

Comment:
Why not keep the existing constructors so that we don't need to pass the empty list for aliases when there aren't any?

Reply:
I chose not to keep the existing constructors to force consideration of aliases and to reduce the amount of overloads. If the old constructors are still desired, I can add them back, but I thought it made more sense to just add aliases, or the empty vector, to everything.